sbncode/BeamSpillInfoRetriever/getFOM.cpp

```diff
@@ -0,0 +1,410 @@
+/**
+ * @file sbncode/BeamSpillInfoRetriever/getFOM.cpp
+ * @brief Figure of Merit for BNB Spills using SBND information adapted from MicroBooNE FOM
+ * @author Max Dubnowski (maxdub@sas.upenn.edu or `@Max Dubnowski` on SBN Slack)
+ */
+#include <math.h>
```

Comment:
The header matching the source file _must_ be always included, usually as the very first `#include`:
```suggestion
#include "sbncode/BeamSpillInfoRetriever/getFOM.h"
#include <math.h>
```